discovery-infra/day2.py

```diff
@@ -140,6 +144,11 @@ def day2_nodes_flow(client,
         )
         log.info("%d worker nodes were successfully added to OCP cluster", num_worker_nodes)
 
+def set_hostnames_from_tf(client, cluster_id, tf_folder, network_name):
+    tf = terraform_utils.TerraformUtils(working_dir=tf_folder)
+    libvirt_nodes = utils.extract_nodes_from_tf_state(tf.get_state(), (network_name), consts.NodeRoles.WORKER)
+    utils.update_hosts(client, cluster_id, libvirt_nodes, update_roles=False, update_hostnames=True)
```

Comment:
i think you should add a comment here, that even if 1st dat cluster had worker nodes installed, the update_hosts function will ignore them , since they are not present in the inventory of day2 cluster